installers/raspberry.sh

```diff
@@ -36,6 +36,10 @@ fi
 function version_gt() { test "$(echo "$@" | tr " " "\n" | sort -V | head -n 1)" != "$1"; }
 function command_exists () { type "$1" &> /dev/null ;}
 
+# Update before first apt-get
+echo -e "\e[96mUpdating packages ...\e[90m"
+sudo apt-get update || exit
```

Comment:
You are right. Just copied the next-line `install`, but it is conceivable the scenario where the packages are installed, the `update` may fail but the overall `install` process may work.